admin/commands/list.go

```diff
@@ -65,7 +65,9 @@ func (a listResultAgent) NiceAgentStatus() string {
 	if res == "" {
 		res = "unknown" //nolint:goconst
 	}
-	res = cases.Title(language.English).String(strings.ToLower(res))
+	res = strings.ToLower(res)
+	res = cases.Title(language.English).String(res)
+	res = strings.ReplaceAll(res, "_", " ")
```

Comment:
Maybe it would be good to extract this into method since it is used in two places. With that we can ensure always same "formatting" in both methods.